uiowa/uiowa
# Footer Block Edit Link

Resolves  #2201

## To Test

Pull and visit /admin/config/system/site-information on a couple uids_base sites. Early collegiate sites didn't have the footer contact block as id = 1 so you could pull down the homepage, etc.
Regardless of what site, as a webmaster you should be able to see the link above footer menus, click it and be taken to the footer contact block that serves content to the footer of the website.

This allows webmasters to get to the footer block even if the contextual filter link on the frontend is missing (empty block).

## Follow-up

Update footer documentation. https://sitenow.uiowa.edu/documentation/basic-site-settings-and-footer-configuration
